databox-install-component

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+## check we have the tools
+if ! [ -x "$(command -v git)" ]; then
+  die 1 "git is not installed;"`
+       `"see https://git-scm.com/"
+fi
+
+if ! [ -x "$(command -v curl)" ]; then
+  die 1 "curl is not installed;"`
+       `"https://curl.haxx.se"
+fi
+
+#include helper functions
+source ./scripts/utils
+
+function fail {
+    echo -e "[$(datef) $ME]: ${1} \nERROR: ${2} $(red FAILED)"
+    exit 1
+}
+
+function success {
+    echo -e "[$(datef) $ME]: ${1} $(green OK)"
+}
+
+function test_assert {
+  if [ "$1" != "$2" ]
+  then
+    fail "$3" "$1"
+  else
+    success "$3"
+  fi
+}
+
```

Comment:
the above three might all move to `scripts/utils` perhaps?
(and I think convention would be that "library script files" get a `.sh` extension; commands don't)